covasim/webapp/cova_app.js

```diff
@@ -235,7 +251,7 @@ var vm = new Vue({
             this.int_pars[key].push(intervention);
             const result = this.int_pars[key].sort((a, b) => a.start - b.start);
             this.$set(this.int_pars, key, result);
-            const response = await sciris.rpc('get_gantt', undefined, {int_pars: this.int_pars, intervention_config: this.interventionTableConfig});
+            const response = await sciris.rpc('get_gantt', undefined, {int_pars: this.int_pars, intervention_config: this.interventionTableConfig, n_days: this.sim_length.best});
```

Comment:
Show the entire sim period, makes it easier to see how long the interventions are actually for